### PR TITLE
Add all missing block replacers

### PR DESCRIPTION
**Pull request by Niko-sk2x**
Submitting PR so it can be reviewed

### CODE REVIEW

File: src/main/java/cubicchunks/worldgen/generator/custom/biome/replacer/SwampWaterWithLilypadReplacer.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        return new IBiomeBlockReplacerProvider() {
+
+            private final ResourceLocation OCEAN_LEVEL = CubicChunks.location("water_level");
+
+            @Override public IBiomeBlockReplacer create(ICubicWorld world, CubicBiome biome, BiomeBlockReplacerConfig conf) {
+                return new SwampWaterWithLilypadReplacer(Biome.GRASS_COLOR_NOISE, MathHelper.floor(conf.getDouble(OCEAN_LEVEL)));
+            }
+
+            @Override public Set<ConfigOptionInfo> getPossibleConfigOptions() {
+                return new HashSet<>();
```
Comment (kashike) [inline src/main/java/cubicchunks/worldgen/generator/custom/biome/replacer/SwampWaterWithLilypadReplacer.java]: `Collections.emptySet()` (or `ImmutableSet.of()`)

Comment (Niko-sk2x) [reply src/main/java/cubicchunks/worldgen/generator/custom/biome/replacer/SwampWaterWithLilypadReplacer.java]: Why? It's used once when loading a world, and never again for a given world. So it's not a real performance improvement, and it's also not a real readability improvement.

Comment (kashike) [reply src/main/java/cubicchunks/worldgen/generator/custom/biome/replacer/SwampWaterWithLilypadReplacer.java]: Useless creation of a `HashSet`& `HashMap`.

Comment (Niko-sk2x) [reply src/main/java/cubicchunks/worldgen/generator/custom/biome/replacer/SwampWaterWithLilypadReplacer.java]: I will wait for a full review and fix more of these small issues at once

Comment (Cyclonit) [reply src/main/java/cubicchunks/worldgen/generator/custom/biome/replacer/SwampWaterWithLilypadReplacer.java]: `Collections.emptySet()` seems to be the most elegant version. I consider it to be a readability improvement.

**Review by Cyclonit — COMMENTED** [review]
Aside from @kashike's point and the rather interesting PR title, I think its fine.

**Comment by Niko-sk2x** [comment]
Fixed PR title... I suck at typing

**Comment by Niko-sk2x** [comment]
Damn, I got commit message wrong... I will probably squash it when merging and fix it there

**Pull request merged**